### PR TITLE
[14.0] [mig] pos_cash_move_reason

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
Is it possible to get the review here?

**Comment by etobella** [comment]
@dsolanki-initos Travis is failing, fix it before asking for reviews please

**Comment by etobella** [comment]
Also, runbot is failing
imagen

**Comment by dsolanki-initos** [comment]
Hi @etobella
I have fix the above error you can check now.

**Comment by etobella** [comment]
now it works as expected, thanks, however, some tests would be great.

**Review by fshah-initos — APPROVED** [review]
LGTM. Functional tests and code review.

**Comment by dsolanki-initos** [comment]
@chienandalu if would be great if you could merge the PR

**Comment by dsolanki-initos** [comment]
@legalsylvain It would be great if you add the reviews here

**Comment by dsolanki-initos** [comment]
It would be great if we cousl get the reviews here

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py [hunk 1]
```diff
@@ -22,6 +23,10 @@ def setUpClass(cls):
         )[0]
         cls.deposit_reason = cls.env.ref("pos_cash_move_reason.bank_out_reason")
 
+    def test_onchange_expense_reason(self):
+        self.deposit_reason._onchange_is_expense_reason()
```
Comment (etobella) [inline pos_cash_move_reason/tests/test_pos_cash_move_reason.py]: The way to test onchanges is to use a Form

Comment (dsolanki-initos) [reply pos_cash_move_reason/tests/test_pos_cash_move_reason.py]: Hi @etobella 
I've updated the test for onchange using Form. You can review it.

**Comment by dsolanki-initos** [comment]
It would be great if we get the reviews here

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**